docs/docsite/rst/roadmap/COLLECTIONS_2_10.rst

```diff
@@ -0,0 +1,39 @@
+====================
+Ansible project 2.10
+====================
+
+This release schedule includes dates for the `ansible <https://pypi.org/project/ansible/>`_ package, with a few dates for the `ansible-base <https://pypi.org/project/ansible-base/>` package as well. All dates are subject to change, please see :ref:`base_roadmap_2_10` for the most recent updates on ansible-base.
+
+.. contents::
+   :local:
+
+Release Schedule
+----------------
+
+.. note:: Dates subject to change.
+.. note:: We plan to post weekly alpha releases to the `PyPI ansible project <https://pypi.org/project/ansible/>`_ for testing.
+
+- 2020-06-23: ansible-2.10 alpha freeze.
+  No net new collections will be added to the ``ansible-2.10`` package after this date.
+- 2020-07-10: Ansible collections freeze date for content shuffling.
+  Content should be in its final collection for the ansible-2.10 release.
+- 2020-08-13: ansible-base 2.10 Release date, see :ref:`base_roadmap_2_10`.
+- 2020-08-14: final ansible-2.10 alpha.
+- 2020-08-18: Ansible 2.10 beta freeze.
+  * No new modules or major features will be added after this date. In practice this means we will freeze the semver collection versions to compatible release versions. For example, if the version of community.crypto on this date was community-crypto-1.1.0; ansible-2.10.0 could ship with community-crypto-1.1.1.  It would not ship with community-crypto-1.2.0.
```

Comment:
```suggestion
  - No new modules or major features will be added after this date. In practice this means we will freeze the semver collection versions to compatible release versions. For example, if the version of community.crypto on this date was community-crypto-1.1.0; ansible-2.10.0 could ship with community-crypto-1.1.1.  It would not ship with community-crypto-1.2.0.
```
With `*` no sublist is started, and the layout looks strange. Maybe `-` works better? (https://github.com/ansible/ansible/blob/308dab40ea25e996a5e2a90ccf7d92eb05143103/docs/docsite/rst/roadmap/COLLECTIONS_2_10.rst)

Same below.

Reply:
Addressed in ebeebfdae26cf43fa99dcf178138b5251dcb3b85. Thanks @felixfontein!